Arduino/Ground Systems/Controls/controls.ino

```diff
@@ -44,15 +46,15 @@ void loop()
         {
           if (command[j] == '1')
           {
-            io.digitalWrite(i, LOW);
+            io.digitalWrite(j-1, 0);
```

Comment:
Same with this, idk what it means

Reply:
This was the key bug; if we use `i`, then we'd be setting the wrong relay with the `ON`/`OFF` when attempting to iterate through each relay. This can be confirmed experimentally or by tracing the source code.

Additionally, using `0` instead of `LOW` is a trivial change; Not required, but I decided to do so because I personally prefer the explicit value for this use case (makes more sense as ON/OFF).